python/lsst/sims/coordUtils/LsstCameraUtils.py

```diff
@@ -108,8 +107,13 @@ def _findDetectorsListLSST(pupilPointList, detectorList, allow_multiple_chips=Fa
 
     @param[in] pupilPointList  a list of points in PUPIL/FIELD_ANGLE coordinates
 
-    @param[in] detecorList is a list of lists.  Each row contains the detectors that should be searched
-    for the correspdonding pupilPoint
+    @param[in] detecorList is a list of the afwCameraGeom detector objects being considered
+
+    @param[in] possible_points is a list of lists.  possible_points[ii] is a list of integers
+    corresponding to the pupil points that may be on detectorList[ii].
+
```

Comment:
After going through the code, maybe it would be clearer to say "corresponding to the indices in pupilPointList of the pupil points that may be ..."?